rmw_connextdds/src/rmw_api_impl_ndds.cpp

```diff
@@ -1037,11 +1037,13 @@ rmw_feature_supported(rmw_feature_t feature)
       {
         return true;
       }
-    default:
+    case RMW_MIDDLEWARE_SUPPORTS_TYPE_DISCOVERY:
```

Comment:
Apply this change in other places.
```suggestion
    case RMW_MIDDLEWARE_SUPPORTS_TYPE_DISCOVERY: [[fallthrough]]
```

Reply:
good practice to let the compiler know we are NOT missing `break` in this case. thanks 👍